Libraries/D3D12RaytracingFallback/src/TreeletReorder.hlsl

```diff
@@ -59,7 +59,7 @@ void FormTreelet(in uint groupThreadId)
             {
                 uint treeletNodeIndex = treeletToReorder[i];
                 // Leaf nodes can't be split so skip these
-                if (!IsLeaf(treeletNodeIndex))
+                if (!IsLeafIndex(treeletNodeIndex))
```

Comment:
IMO IsLeaf is better as it's easier to read and comprehend. I don't see it being necessary to stress the index part. I could see an argument for IsLeafIndex() if there was IsLeaf() for an object reference/pointer, but that's not the case and even then IsLeaf should be clear enough.

Reply:
That makes sense.